signup/api/users.py

```diff
@@ -75,7 +75,7 @@ def perform_update(self, serializer):
             # except the current one if
             # django.contrib.auth.middleware.SessionAuthenticationMiddleware
             # is enabled.
-            update_session_auth_hash(self.request, serializer.instance)
+            update_session_auth_hash(self.request, self.request.user)
```

Comment:
`serializer.instance` worked. The idea was that it might not be necessary to call `update_session_auth_hash` unless the password is updated for the `request.user`. My guess is that with `update_session_auth_hash(self.request, serializer.instance)` the user gets logged out as soon as the manager (`request.user`) updates the password for the user (`serializer.instance`), ... or nothing happens.

Not a big deal. Just worth understanding why the code is there in the first place.